app/bundles/NotificationBundle/Config/config.php

```diff
@@ -146,6 +146,14 @@
                 ],
             ],
         ],
+        'integrations' => [
```

Comment:
@mqueme Here is an example of how to register the integrations in your bundle config.